src/openzaak/components/zaken/tests/test_notifications_send.py

```diff
@@ -627,3 +632,82 @@ def test_zaakbesluit_create_fail_send_notification_create_db_entry(self):
 
         self.assertEqual(failed.statuslog_ptr, logged_warning)
         self.assertEqual(failed.message, message)
+
+
+@override_settings(NOTIFICATIONS_DISABLED=False)
+class InvalidNotifConfigTests(JWTAuthMixin, APITransactionTestCase):
+
+    client_id = "test"
+    secret = "test"
+    heeft_alle_autorisaties = True
+
+    def setUp(self):
+        JWTSecret.objects.get_or_create(
+            identifier=self.client_id, defaults={"secret": self.secret}
+        )
+
+        self.applicatie = Applicatie.objects.create(
+            client_ids=[self.client_id],
+            label="for test",
+            heeft_alle_autorisaties=self.heeft_alle_autorisaties,
+        )
+
+        super().setUp()
+
+    def test_invalid_notification_config_create(self):
+        conf = NotificationsConfig.get_solo()
+        conf.api_root = "bla"
+        conf.save()
+
+        url = get_operation_url("zaak_create")
+        zaaktype = ZaakTypeFactory.create(concept=False)
+        zaaktype_url = reverse(zaaktype)
+        data = {
+            "zaaktype": f"http://testserver{zaaktype_url}",
+            "vertrouwelijkheidaanduiding": VertrouwelijkheidsAanduiding.openbaar,
+            "bronorganisatie": "517439943",
+            "verantwoordelijkeOrganisatie": VERANTWOORDELIJKE_ORGANISATIE,
+            "registratiedatum": "2012-01-13",
+            "startdatum": "2012-01-13",
+            "toelichting": "Een stel dronken toeristen speelt versterkte "
+            "muziek af vanuit een gehuurde boot.",
+            "zaakgeometrie": {
+                "type": "Point",
+                "coordinates": [4.910649523925713, 52.37240093589432],
+            },
+        }
+
+        # with self.assertRaises(KeyError):
+        self.client.post(url, data, **ZAAK_WRITE_KWARGS)
+
+        self.assertFalse(Zaak.objects.exists())
+        self.assertFalse(StatusLog.objects.exists())
+
+    def test_notification_config_inaccessible_create(self):
+        conf = NotificationsConfig.get_solo()
+        conf.api_root = "http://localhost:8001/api/v1/"
+        conf.save()
+
+        url = get_operation_url("zaak_create")
+        zaaktype = ZaakTypeFactory.create(concept=False)
+        zaaktype_url = reverse(zaaktype)
+        data = {
+            "zaaktype": f"http://testserver{zaaktype_url}",
+            "vertrouwelijkheidaanduiding": VertrouwelijkheidsAanduiding.openbaar,
+            "bronorganisatie": "517439943",
+            "verantwoordelijkeOrganisatie": VERANTWOORDELIJKE_ORGANISATIE,
+            "registratiedatum": "2012-01-13",
+            "startdatum": "2012-01-13",
+            "toelichting": "Een stel dronken toeristen speelt versterkte "
+            "muziek af vanuit een gehuurde boot.",
+            "zaakgeometrie": {
+                "type": "Point",
+                "coordinates": [4.910649523925713, 52.37240093589432],
+            },
+        }
+
+        # with self.assertRaises(ConnectionError):
```

Comment:
@sergei-maertens maybe you know why, but for some reason this fails in travis. The exception does show up in the logs, but the assertion fails